### PR TITLE
creating environment class and edit in room class

### PR DESCRIPTION
**Pull request by viktoriaNadarajah**

### CODE REVIEW

File: src/worldofzuul/Environment.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+package worldofzuul;
+
+import java.util.HashMap;
+
+/**
+ *
+ * @author ViktoriaNadarajah
+ */
+public class Environment {
+    private HashMap<String, Room> rooms;
```
Comment (quartercastle) [inline src/worldofzuul/Environment.java]: Missing `docblock` comment. 😄

Comment (quartercastle) [reply src/worldofzuul/Environment.java]: Both for attributes and methods.

File: src/worldofzuul/Room.java [hunk 2]
```diff
@@ -53,17 +69,17 @@ public String getShortDescription() {
...
-        Set<String> keys = exits.keySet();
-        for(String exit : keys) {
-            returnString += " " + exit;
-        }
-        return returnString;
+    /**
+     * Gets the exit for the current room
+     * @return the exits from the room 
+     */
+    private String getExitString() {
```
Comment (quartercastle) [inline src/worldofzuul/Room.java:76]: Should be spaces instead of tab.

File: src/worldofzuul/Room.java [hunk 3]
```diff
@@ -21,6 +27,16 @@
     private HashMap<String, Room> exits;
 
     /**
+     * The ArrayList keeps the items
+     */
+    private ArrayList<Item> items;
```
Comment (quartercastle) [inline src/worldofzuul/Room.java]: Think an array would be better here.

File: src/worldofzuul/Room.java [hunk 4]
```diff
@@ -74,4 +90,30 @@ private String getExitString() {
...
+    }
+    
+    /**
+     * This method adds items to our ArrayList
+     * @param items
+     * @return returns an item into our ArrayList
+     */
+    public boolean addItems(ArrayList items) {
+       return items.add(item);
+    }
```
Comment (quartercastle) [inline src/worldofzuul/Room.java]: If we change the items type from `ArrayList` to `Array`, we can more easily set the items with the array shorthand like below.
```java
room.addItems({
     new Ketchup(),
     new Computer()
})
```

File: src/worldofzuul/Environment.java (hunk 1 above)
Comment (viktoriaNadarajah) [reply src/worldofzuul/Environment.java]: fixed in: 514c5e8cab8545e8bab67a5dde687ac8426441fa

File: src/worldofzuul/Room.java (hunk 3 above)
Comment (viktoriaNadarajah) [reply src/worldofzuul/Room.java]: fixed in: 718f5ca2b3d2a167093b40a4fc2b4826fc510239

File: src/worldofzuul/Room.java (hunk 2 above)
Comment (viktoriaNadarajah) [reply src/worldofzuul/Room.java:76]: fixed in: 718f5ca2b3d2a167093b40a4fc2b4826fc510239

File: src/worldofzuul/Room.java (hunk 4 above)
Comment (viktoriaNadarajah) [reply src/worldofzuul/Room.java]: fixed in: 718f5ca2b3d2a167093b40a4fc2b4826fc510239

**Review by quartercastle — APPROVED** [review]
👍

**Pull request merged**